python/ray/data/preprocessors/concatenator.py

```diff
@@ -0,0 +1,80 @@
+from typing import List, Optional
+import numpy as np
+import pandas as pd
+
+from ray.data.extensions import TensorArray
+from ray.data.preprocessor import Preprocessor
+
+
+class Concatenator(Preprocessor):
+    """Create tensor columns via concatenation.
+
+    A tensor column is a a column consisting of ndarrays as elements.
+
+    Example:
+        >>> import pandas as pd
+        >>> from ray.data.preprocessors import Concatenator
```

Comment:
```suggestion
        >>> from ray.data.preprocessors import Concatenate
```

Prefer the verb instead of the noun form for preprocessors.

Reply:
Hmm, this would make it inconsistent with all of our other preprocessors.

Further, other ML frameworks (huggingface, scikit-learn, spark) have adopted the noun form for preprocessors too -- https://spark.apache.org/docs/latest/api/python/reference/pyspark.ml.html#feature

Reply:
OK